agreement_maintenance/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2018 Pavlov Media
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Maintenance Agreements",
+    "summary": "Manage maintenance agreements and contracts",
+    "author": "Pavlov Media, "
+    "Open Source Integrators, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "category": "Maintenance",
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "depends": ["maintenance", "agreement_serviceprofile"],
```

Comment:
travis has problem with agreement_serviceprofile module, probably it is not in the 13.0 repo yet?

Reply:
Well, this PR depends on the migration (#760)